recipes-bsp/tegra-binaries/tegra-helper-scripts/initrd-flash.sh

```diff
@@ -243,6 +243,7 @@ sign_binaries() {
     fi
     if [ "$CHIPID" = "0x18" ]; then
 	"$here/tegra-signimage-helper" --chip "$CHIPID" --nosplit --type kernel initrd-flash.img
+	"$here/tegra-signimage-helper" --chip "$CHIPID" --nosplit --type kernel-dtb ${INITRD_DTBFILE}
```

Comment:
Let's let INITRD_DTBFILE be null if there is no separate device tree defined for initrd flashing, and just add a check for it being non-null here to see if the extra signing is needed.